### DISCUSSION TITLE
How can I remove the remote path from the statusline?

### DISCUSSION THREAD

**Discussion by micdonato** [vscode-neovim/vscode-neovim#2209] [Q&A] [answered]
Not sure if this is the right place, let me know if there are other places for these kind of questions

I only work on remote tunnels and dev containers, and the file path in the status line is usually humongous, especially in dev containers.

Does anyone have any suggestion on how to trim it to something manageable?

This is the example of a devcontainer:
<img width="781" alt="Screenshot 2024-08-21 at 3 57 44 PM" src="https://github.com/user-attachments/assets/476fc5f6-7288-4bbc-b698-9a3c410db138">

In remote tunnels I can usually see the file name if I remove a bunch of other elements from VS Code's bar, but it is still pretty long

**Comment by justinmk** [comment, marked as answer]
You have full control over 'statusline'. That doesn't look like the default, though. Did you set 'statusline' or do you have a statusline plugin installed? 

Does it happen if you [set `vscode-neovim.neovimClean`](https://github.com/vscode-neovim/vscode-neovim) ? 

Try `:set statusline?` to print the value of 'statusline' option. If it's not empty, then your config has set it somewhere.

**Reply by micdonato** [reply]
So, I didn’t know if it was an nvim issue because I was only seeing it on the remote connection when I used vscode (no idea how to connect to a kubernetes pod via nvim).

I tried to do `:set statusline ` from vscode but I was not getting anything.

I did it from nvim and it turns out it was dealt with by mini.statusline (I used kickstart-neovim and I have no idea that was installed).

I then asked ChatGPT to help me with it and now I shortened it to only the last three directories.

thanks!
